src/plugins/controls/public/options_list/components/options_list_popover.tsx

```diff
@@ -43,36 +41,33 @@ export const OptionsListPopover = ({
   const [showOnlySelected, setShowOnlySelected] = useState(false);
 
   return (
-    <>
+    <div
```

Comment:
I just removed the wrapping fragment, since it's unnecessary - no other changes were made here.